### PR TITLE
fix: update CRUD for fashion example

### PR DESCRIPTION
**Pull request by rutujasurve94**
update Jina to 1.1.0

modify flow yamls to enable REST API

add index_rest that indexes docs using the /index REST API

add index_rest to click options

change README to point towards /search (NOT /api/search)

### CODE REVIEW

File: fashion-example-query/app.py [hunk 1]
```diff
@@ -154,6 +153,39 @@ def query(num_doc, target: dict):
...
+        print(f'Indexing {data_path}')
+        url = f'http://0.0.0.0:{f.port_expose}/index'
+
+        input_docs = _input_lines(
+            filepath=data_path,
+            size=num_docs,
+            read_mode='r',
+        )
+        data_json = {'data': [Document(text=text).dict() for text in input_docs]}
+        print(f'#### {len(data_json["data"])}')
```
Comment (cristianmtr) [inline fashion-example-query/app.py:170]: remove print

File: fashion-example-query/app.py [hunk 2]
```diff
@@ -191,6 +224,19 @@ def main(task, num_docs_query, num_docs_index):
...
+            print(
+                f'\n +------------------------------------------------------------------------------------+ \
+                    \n |                                   🤖🤖🤖                                           | \
+                    \n | The directory {workspace} already exists. Please remove it before indexing again.  | \
+                    \n |                                   🤖🤖🤖                                           | \
+                    \n +------------------------------------------------------------------------------------+'
+            )
+            sys.exit(1)
+
+    print(f'### task = {task}')
```
Comment (cristianmtr) [inline fashion-example-query/app.py:239]: remove print

**Review by cristianmtr — DISMISSED** [review]

**Comment by cristianmtr** [comment]
Link it to the main ticket, if you haven't

File: fashion-example-query/app.py [hunk 3]
```diff
@@ -154,6 +153,38 @@ def query(num_doc, target: dict):
     write_html(os.path.join('./workspace', 'hello-world.html'))
 
 
+def index_restful(num_docs):
+    f = Flow().load_config('flows/index.yml')
+
+    with f:
+        data_path = os.path.join(os.path.dirname(__file__), os.environ.get('JINA_DATA_FILE', None))
+        print(f'Indexing {data_path}')
```
Comment (cristianmtr) [inline fashion-example-query/app.py:161]: remove print

**Review by cristianmtr — CHANGES_REQUESTED** [review]
remove print

File: fashion-example-query/app.py (hunk 3 above)
Comment (rutujasurve94) [reply fashion-example-query/app.py]: done

**Comment by nan-wang** [comment]
This PR needs further improvement and we are working on `1.2` anyway.

**Pull request closed without merging**